### PR TITLE
Move debug output to STDERR

### LINKED ISSUE #1542: Debug info should go to stderr
```
$ DEBUG=true kontena vpn config > foo.ovpn
$ head foo.ovpn
D, [2016-12-12T10:30:25.756673 #51676] DEBUG -- CONFIG: Loading configuration from /Users/kimmo/.kontena_client.json
D, [2016-12-12T10:30:25.758316 #51676] DEBUG -- CONFIG: Configuration loaded with 18 servers.
D, [2016-12-12T10:30:25.758356 #51676] DEBUG -- CONFIG: Current master: dry-dream-74
...
```

### PR DESCRIPTION
**Pull request by kke**
Fixes #1542

### CODE REVIEW

File: cli/lib/kontena/cli/common.rb [hunk 1]
```diff
@@ -12,7 +12,7 @@ module Common
 
       def logger
         return @logger if @logger
-        @logger = Logger.new(STDOUT)
+        @logger = Logger.new(ENV["DEBUG"] ? STDERR : STDOUT)
```
Comment (SpComb) [inline cli/lib/kontena/cli/common.rb]: Should the Logger just always use `STDERR`?

**Comment by SpComb** [comment]
Could all of the `ENV["DEBUG"] && STDERR.puts(...)` be replaced with `logger.debug(...)` (or `logger.debug { ... }` to keep the lazy evaluation)?

File: cli/lib/kontena/client.rb [hunk 2]
```diff
@@ -46,7 +46,7 @@ def initialize(api_url, token = nil, options = {})
       uri = URI.parse(@api_url)
       @host = uri.host
 
-      @logger = Logger.new(STDOUT)
+      @logger = Logger.new(ENV["DEBUG"] ? STDERR : STDOUT)
```
Comment (SpComb) [inline cli/lib/kontena/client.rb]: At least here this can just be `STDERR`, since there are only `logger.debug ...` calls in the class?

Comment (kke) [inline cli/lib/kontena/client.rb]: All the logging should be unified anyways at some point.

**Comment by timolehto** [comment]
Just a thought, but couldn't you just provide an option for outputting the configs to a file instead of stdout i.e. something like: `kontena vpn config -f kontena.vpn.config`?

**Comment by kke** [comment]
This will also help with piping to other commands when debug=true.

**Comment by timolehto** [comment]
True

**Pull request merged**